config/locales/candidate_interface/submit_application.yml

```diff
@@ -14,6 +14,8 @@ en:
               site_invalid: You cannot submit this application because it’s no longer available. You need to either remove it or change the course.
               course_full: You cannot submit this application because there are no places left on the course.
               remove_or_change_application: You need to either remove this application or change your course.
+              science_gcse_missing: To apply for a Primary course, you need a GCSE in science at grade 4 (C) or above, or equivalent.
+              science_gcse_missing_guide: <a href="%{link}">Add your science GCSE grade (or equivalent) before submitting this application.</a>
```

Comment:
If we're using links we need to apply the `govuk_link_to` styling as there's a subtle difference between links 👇 

without:
<img width="698" alt="image" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/47917431/0e28d433-b83a-4576-8e6c-4f307fce8efc">

with:
<img width="367" alt="image" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/47917431/82a0c3c7-1f0c-415f-a8d4-e08a64399b89">

Reply:
Yeah, had it initially, but that's not going to work nicely with translations. I'll make it work somehow.

Reply:
Also, the full stop at the end of the linked sentence should not be included in the link. I know, I know, I don't make the rules ok!